### PR TITLE
Upgrade operator-sdk and controller-gen versions

### PR DESCRIPTION
**Pull request by kflorianski-ibm**

### CODE REVIEW

File: config/crd/bases/operator.ibm.com_ibmlicensingmetadatas.yaml [hunk 1]
```diff
@@ -19,23 +18,28 @@ spec:
...
-          Processor Core (VPC) metric by Pods that are managed by automation, for
-          which licensing annotations are not defined at the time of deployment, such
-          as the open source OpenLiberty. / For more information, see documentation:
-          https://ibm.biz/icpfs39install. License: By installing this product you
-          accept the license terms. For more information about the license, see https://ibm.biz/icpfs39license.'
+        description: |-
+          IBMLicensingMetadata is the schema for IBM License Service. Thanks to IBMLicensingMetadata, you can track
+          the license usage of Virtual Processor Core (VPC) metric by Pods that are managed by automation,
+          for which licensing annotations are not defined at the time of deployment, such as the open source OpenLiberty.
+          / For more information, see documentation: https://ibm.biz/icpfs39install.
```
Comment (jakubsolecki) [inline config/crd/bases/operator.ibm.com_ibmlicensingmetadatas.yaml:25]: The doc link `https://ibm.biz/icpfs39install` is incorrect here. It points to the LTSR docs, even though it's a CD stream.

**Review by jakubsolecki — APPROVED** [review]
Please check whether text on the OCP UI displays correctly after these changes.

File: config/crd/bases/operator.ibm.com_ibmlicensingmetadatas.yaml (hunk 1 above)
Comment (ljeda) [reply config/crd/bases/operator.ibm.com_ibmlicensingmetadatas.yaml:25]: good catch

**Review by ljeda — APPROVED** [review]
please apply @jakubsolecki's remark about the link and we're good to merge.

**Pull request merged**